### PR TITLE
Fix failing double check test for sass-parser

### PR DESCRIPTION
**Pull request by jathak**

### CODE REVIEW

File: tool/grind/double_check.dart [hunk 1]
```diff
@@ -36,12 +36,15 @@ Future<void> doubleCheckBeforeRelease() async {
...
-      var pubspec = Pubspec.parse(File("$dir/pubspec.yaml").readAsStringSync(),
-          sourceUrl: p.toUri("$dir/pubspec.yaml"));
-
-      var package = await client.packageInfo(pubspec.name);
-      if (pubspec.version == package.latestPubspec.version) {
-        fail("${pubspec.name} ${pubspec.version} has already been released!");
+      var pubspecFile = File("$dir/pubspec.yaml");
+      if (pubspecFile.existsSync()) {
+        var pubspec = Pubspec.parse(pubspecFile.readAsStringSync(),
+            sourceUrl: p.toUri("$dir/pubspec.yaml"));
```
Comment (ntkme) [inline tool/grind/double_check.dart:42]:
```suggestion
            sourceUrl: pubspecFile.uri);
```
This also allows removal of `import 'package:path/path.dart'`

File: tool/grind/double_check.dart [hunk 2]
```diff
@@ -36,12 +36,15 @@ Future<void> doubleCheckBeforeRelease() async {
...
       ...Directory("pkg").listSync().map((entry) => entry.path)
     ]) {
-      var pubspec = Pubspec.parse(File("$dir/pubspec.yaml").readAsStringSync(),
-          sourceUrl: p.toUri("$dir/pubspec.yaml"));
-
-      var package = await client.packageInfo(pubspec.name);
-      if (pubspec.version == package.latestPubspec.version) {
-        fail("${pubspec.name} ${pubspec.version} has already been released!");
+      var pubspecFile = File("$dir/pubspec.yaml");
+      if (pubspecFile.existsSync()) {
```
Comment (nex3) [inline tool/grind/double_check.dart:40]: Nit: avoid excess indentation via
```suggestion
      if (!pubspecFile.existsSync()) continue;
```

**Review by nex3 — APPROVED** [review]

File: pubspec.yaml [hunk 3]
```diff
@@ -1,5 +1,5 @@
 name: sass
-version: 1.78.0
+version: 1.78.1
```
Comment (ntkme) [inline pubspec.yaml:2]: Since nothing has actually been release, we could remove the git tag, and then retag the same release.

Comment (nex3) [reply pubspec.yaml:2]: I agree

**Pull request merged**